### PR TITLE
Propose to assess AWS Lambda.

### PR DESCRIPTION
**Pull request by andreyk0**

### CODE REVIEW

File: recommendations/Deployment.md [hunk 1]
```diff
@@ -8,6 +8,11 @@
     Docker is largely in use as a replacement for RPMs, to wrap a program in some easily deployable format.
     We believe the ability to specify runtime requirements at development time, and not having to worry about the runtime capabilities of the server is extremely valuable.
 
+  - [AWS Lambda](http://aws.amazon.com/lambda/)
+
```
Comment (Seanstoppable) [inline recommendations/Deployment.md]: I would rather Lambda be in trial. I feel like we need to figure out good use cases.

For example, I have seen someone use it for filtering SNS messages and dumping appropriate ones in a queue, where I feel that is kinda overkill and leads to more complex designs needlessly.

Comment (andreyk0) [inline recommendations/Deployment.md]: How'd you do it in this example?

Comment (Seanstoppable) [inline recommendations/Deployment.md]: Oh, actually, I think this is weird in 'Deployment' as well. What are you using it for in this context? If you can fill in what 'usage' of it we feel should be adopted, that would be better.

Comment (andreyk0) [inline recommendations/Deployment.md]: This is what it currently says: " A nice alternative to deploying/managing service clusters.
Consider using in places where functionality is very simple, especially if the traffic is low and it's hard to justify the cost of running EC2 nodes all the time. "

I had some difficulty finding a good category for it but, in the end, it *is* about deploying a tiny bit of functionality in a lightweight form.

Comment (Seanstoppable) [inline recommendations/Deployment.md]: I mean, technically it is a piece of code that is being executed vs some tools we use for managing deployments. More of a framework maybe?

Either way, I feel like we should have a success story/good usage examples at Gilt before it becomes 'adopt'

Comment (zcohn) [inline recommendations/Deployment.md]: Seems reasonable to me to put it in deployment, and also to have some usage before it is adopted.

Comment (gheine) [inline recommendations/Deployment.md]: :+1: Feels like it belongs both into the deployment as well as the frameworks category

**Comment by trenaman** [comment]
+1 (is that how we're doing this - just +1 on github?)

**Comment by gheine** [comment]
:+1:

**Comment by gheine** [comment]
Voted For: 8, Against: 0, Abstained: 0

**Comment by trenaman** [comment]
Currently being assessed by interns (Team Rookie) - look forward to hearing more and potentially moving to Trial or Adopt later.

**Pull request merged**